workshops/geoserver/intro/doc/source/install/suite.rst

```diff
@@ -0,0 +1,230 @@
+.. _install.suite:
+
+Boundless Suite GeoServer Distribution
+======================================
+
+Boundless Suite is a complete web-based geospatial software stack. In this package, the applications contained are:
+
+* **PostGIS** - A spatially enabled object-relational database.
+* **GeoServer** - A software server for loading and sharing geospatial data.
+* **GeoWebCache** - A tile cache server that accelerates the serving of maps (built into GeoServer).
+* **OpenLayers** - A browser-based mapping framework
+
+.. figure:: img/ecosystem.png
+
+   Boundless Suite components and ecosystem
+
+Boundless Suite is free and open source, and is available for download from `Boundless <http://boundlessgeo.com/>`_. Boundless Suite customers are provided with the latest release along with commercial training and support. Boundless Suite includes slightly different extensions than the community releases of GeoServer. Boundless Suite components are subject to extensive integration testing and are available in a range of packages from virtual machines through to linux packages, web archives and cloud/clustering options.

```

Comment:
Suggest: "Boundless Suite components are subject to extensive integration testing and are available in a range of packages including virtual machines, Linux packages, web archives, and cloud/clustering options."